setup.py

```diff
@@ -6,6 +6,9 @@
 
 VERSION = '2.7.2'
 
+with open('README.rst', encoding='utf-8') as f:
```

Comment:
@wesinator Would you mind updating this to be conditional, similar to how this is done?
https://github.com/vertexproject/common-tools/blob/master/setup.py#L21

Reply:
and set the type so it is not ambiguous